CONTRIBUTING.md

```diff
@@ -64,7 +64,7 @@ From the top-level xrpl.js folder (one level above `packages`), run the followin
 ```bash
 npm install
 # sets up the rippled standalone Docker container - you can skip this step if you already have it set up
-docker run -p 6006:6006 --interactive -t --volume $PWD/.ci-config:/config/ xrpllabsofficial/xrpld:1.12.0-b1 -a --start
+docker run -p 6006:6006 --interactive -t --volume $PWD/.ci-config:/opt/ripple/etc/ --platform linux/amd64 rippleci/rippled:2.0.0-b3 /opt/ripple/bin/rippled -a --conf /opt/ripple/etc/rippled.cfg
```

Comment:
Do we need to specify 2.0.0-b3 or can we specify an image that we consistently update with the latest version? (like `:latest` with Weitse's repo?)

afaik we only specified a version specifically because we needed a custom release that wasn't an official release, but if it's our docker container we can have a :develop image potentially that doesn't have the same "release only" requirement

Reply:
As of now there is no `latest` tag, so the version does need to be specified: https://hub.docker.com/r/rippleci/rippled/tags